client/client.go

```diff
@@ -51,15 +51,21 @@ func (c Client) withProject(project string) *Client {
 func Configure(logger hclog.Logger, config interface{}) (schema.ClientMeta, error) {
 	providerConfig := config.(*Config)
 	projects := providerConfig.ProjectIDs
-	serviceAccountKeyJSON := []byte(providerConfig.ServiceAccountKeyJSON)
 
+	serviceAccountKeyJSON := []byte(providerConfig.ServiceAccountKeyJSON)
 	if len(serviceAccountKeyJSON) == 0 {
 		serviceAccountKeyJSON = []byte(os.Getenv(serviceAccountEnvKey))
 	}
 
+	// Add a fake request reason because it is not possible to pass nil options
+	options := []option.ClientOption{option.WithRequestReason("cloudquery resource fetch")}
```

Comment:
I am assuming this gets logged somewhere in GCP, do we want to put in the provider version as well?

Reply:
Good idea, but `Version` is in the `provider` package so accessing it here creates a cyclic import. So, either `Version` needs to move to `client` or `Configure()` can return a closure, with Version enclosed... Room for discussion I think